src/components/repositories/RepositoryTable.tsx

```diff
@@ -0,0 +1,114 @@
+import { useMemo } from 'react';
+import { useRecoilValue } from 'recoil';
+import { useNavigate } from 'react-router-dom';
+import { ThemeProvider } from '@emotion/react';
+import { graphql } from 'babel-plugin-relay/macro';
+import { createFragmentContainer } from 'react-relay';
+
+import { createTheme } from '@mui/material/styles';
+import { createStyles, withStyles, WithStyles } from '@mui/styles';
+import { Stack, Table, TableRow, TableHead, TableBody, TableCell, Typography } from '@mui/material';
+
+import { muiThemeOptions } from '../../cirrusTheme';
+import { navigateRepositoryHelper } from '../../utils/navigateHelper';
+import BuildPreview from '../builds/BuildPreview';
+
+import { RepositoryTable_repositories } from './__generated__/RepositoryTable_repositories.graphql';
+
+// todo: move custom values to mui theme adjustments
+const styles = () =>
+  createStyles({
+    row: {
+      cursor: 'pointer',
+    },
+    cellRepository: {},
+    cellLastBuild: {
+      width: '600px',
+      maxWidth: '600px',
+      minWidth: '600px',
+    },
+  });
+
+const styled = withStyles(styles);
+
+interface Props extends WithStyles<typeof styles> {
+  repositories: RepositoryTable_repositories;
+}
+
+const RepositoryTable = styled(({ classes, repositories = [] }: Props) => {
+  const themeOptions = useRecoilValue(muiThemeOptions);
+  const muiTheme = useMemo(() => createTheme(themeOptions), [themeOptions]);
+
+  return (
+    <ThemeProvider theme={muiTheme}>
+      <Table>
+        <TableHead>
+          <HeadRow />
+        </TableHead>
+        <TableBody>
+          {repositories.map(repository => (
+            <RepositoryRow key={repository.id} repository={repository} />
+          ))}
+        </TableBody>
+      </Table>
+    </ThemeProvider>
+  );
+});
+
+interface HeadRowProps extends WithStyles<typeof styles> {}
+
+const HeadRow = styled(({ classes }: HeadRowProps) => {
+  return (
+    <TableRow>
+      <TableCell className={classes.cellRepository}>Repository</TableCell>
```

Comment:
Just a nit: perhaps this should be called `Name` instead of `Repository`, otherwise it sounds a bit tautologic:

<img width="182" alt="Screenshot 2022-12-26 at 16 07 17" src="https://user-images.githubusercontent.com/85709/209547234-0f294851-3ce7-4d2b-8a05-d596e3efc61b.png">

Reply:
Agree with u!